dashboard/app/helpers/users_helper.rb

```diff
@@ -160,6 +195,8 @@ def user_summary(user)
   #   this in advance for many users in some use cases.
   # @param [Enumerable<Integer>] paired_user_levels
   #   A collection of UserLevel ids where the user was pairing.
+  # @param [Boolean] include_timestamp
```

Comment:
are there any cases where we don't pass `true` for this? i can't think of a use case for that. if we're not using it, let's remove it!

Reply:
`true` isn't passed in here https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/helpers/users_helper.rb#L143

Reply:
hmmm. i find that questionable. from a brief inspection it looks like we only include that data for the teacher dashboard, but i can't think of a reason for that. it doesn't require any extra db queries and would result in _very_ minor data bloat, so imo it would be worth the reduction in complexity to remove it. could you try to find out if there's a specific reason we're excluding it?

Reply:
I agree we should investigate, going to punt this for now to avoid increasing the scope of this ticket